packages/server/src/api/controllers/static/index.ts

```diff
@@ -24,14 +23,75 @@ import AWS from "aws-sdk"
 import fs from "fs"
 import sdk from "../../../sdk"
 import * as pro from "@budibase/pro"
-import { App, Ctx, ProcessAttachmentResponse } from "@budibase/types"
+import {
+  UserCtx,
+  App,
+  Ctx,
+  ProcessAttachmentResponse,
+  Feature,
+} from "@budibase/types"
 import {
   getAppMigrationVersion,
   getLatestMigrationId,
 } from "../../../appMigrations"
 
 import send from "koa-send"
 
+const getThemeVariables = (theme: string) => {
```

Comment:
Can we move this into a theme file in `src/constants`, just to avoid polluting the controllers with spectrum info.